package.json

```diff
@@ -71,9 +71,10 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "mocha -t 10000 dist/test",
+    "test": "mocha -t 10000 --exit dist/test",
```

Comment:
Good catch. Didn't realize that got [changed in mocha 4](https://mochajs.org/#--exit----no-exit). The grunt mocha change may need to be changed too.